app/api/routers/breeze_buddy/templates/__init__.py

```
@@ -160,3 +162,42 @@ async def get_template_by_id(
         Complete TemplateModel including flow, schemas, and metadata
     """
     return await get_template_by_id_handler(template_id, current_user)
+
+
+@router.put("/templates/{template_id}", response_model=TemplateModel)
+async def update_template(
+    template_id: str,
+    template_data: UpdateTemplateRequest,
+    current_user: UserInfo = Depends(get_current_user_with_rbac),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)